### ISSUE TITLE
BUG(0.4.9): latest releast breaks during NPM package update/installation, temp solution is to use 0.4.8

### ISSUE THREAD

**Issue by KemingHe** [RealFaviconGenerator/core#7]
# Bug Report

## My ENV

OS: Ubuntu Linux 24.04.1 LTS
Dev: Node 22, PNPM 9.15.0 and NPM 10.9.0 (tested both)
Proj: NextJS when running dev server via `next dev` (doesn't affect issue but just FYI)

## Problem

Realfavicon npm package no longer installs correctly when upgrading to `0.4.9`, resulting in `sh: 1: realfavicon: not found` error when trying to use the buggy realfavicon package. (See raw output at the end @phbernard .)

## Solution (Workaround)

Use last stable release `realfavicon@0.4.8`.

### If used as `npx`

```bash
npx realfavicon@0.4.8 check 3000
```

### If used as dev dependency

```bash
npm install --save-dev realfavicon@0.4.8
# or
yarn add -D realfavicon@0.4.8
# or
pnpm add -D realfavicon@0.4.8
```

**Then make sure to update your `package.json` to restrict `realfavicon` update**:

```json
"devDependencies": {
  "realfavicon": "0.4.8"
}
```

## Raw Console Output

```bash
keminghe@keminghe-virtual-machine:~/Projects/KemingHe/buckeye-gpt$ pn add -D realfavicon@latest
 WARN  6 deprecated subdependencies found: are-we-there-yet@2.0.0, gauge@3.0.2, glob@7.2.3, inflight@1.0.6, npmlog@5.0.1, rimraf@3.0.2
Packages: +1
+
Progress: resolved 1161, reused 1046, downloaded 0, added 0, done
 WARN  Failed to create bin at /home/keminghe/Projects/KemingHe/buckeye-gpt/node_modules/.pnpm/realfavicon@0.4.9_rollup@4.28.1/node_modules/realfavicon/node_modules/.bin/realfavicon. ENOENT: no such file or directory, open '/home/keminghe/Projects/KemingHe/buckeye-gpt/node_modules/.pnpm/realfavicon@0.4.9_rollup@4.28.1/node_modules/realfavicon/dist/index.js'
 WARN  Failed to create bin at /home/keminghe/Projects/KemingHe/buckeye-gpt/node_modules/.bin/realfavicon. ENOENT: no such file or directory, open '/home/keminghe/Projects/KemingHe/buckeye-gpt/node_modules/realfavicon/dist/index.js'

devDependencies:
- realfavicon 0.4.8
+ realfavicon 0.4.9

Done in 13.1s
keminghe@keminghe-virtual-machine:~/Projects/KemingHe/buckeye-gpt$ pn check:favicon

> buckeye-gpt@0.3.1 check:favicon /home/keminghe/Projects/KemingHe/buckeye-gpt
> realfavicon check -s cli 3000

sh: 1: realfavicon: not found
 ELIFECYCLE  Command failed.
```

**Comment by phbernard** [comment]
Damn 😣 Thank you for reporting.

It should be fixed now. Could you give it a try?

**Comment by KemingHe** [comment]
@phbernard Thank you for the awesome fix in e954729 . `realfavicon@0.4.10` now behaves normally.

For future ref, I highly recommend adding automated e2e testing to run before each release. We can def talk more on that if you're interested (I'll open another issue.)

Otherwise you can mark this issue as resolved. 🌻
